src/tests/suite/daos_test_common.c

```diff
@@ -105,6 +105,7 @@ test_setup_pool_create(void **state, struct test_pool *ipool,
 		else
 			print_message("setup: created pool "DF_UUIDF"\n",
 				       DP_UUID(outpool->pool_uuid));
+		uuid_unparse(outpool->pool_uuid, outpool->pool_str);
```

Comment:
so why not just switch to using labels in the entire testsuite instead of uuid_str?